### DISCUSSION TITLE
sort icon and position in AnalyticalTable

### DISCUSSION THREAD

**Discussion by giladvidal** [UI5/webcomponents-react#2969] [Q&A] [answered]
Hi,
How is it possible to set the sort icon on the left? set other icon?

Thanks,
Gilad

**Comment by Lukas742** [comment, marked as answer]
Hi @giladvidal 

this behavior is not supported out of the box. To achieve this you will have to implement a custom header component and add the icons yourself there. Also you have to make sure that the internal icon container div is not displayed. The container is currently not easily accessible, but we'll add a `data-component-name` data attribute [in our next release](https://github.com/SAP/ui5-webcomponents-react/pull/2983). For now you could implement it like this: [codeSandbox](https://codesandbox.io/s/defaulttable-forked-vdyopo?file=/src/App.js)
